### PR TITLE
Updated the root package.json to prevent npm install errors, updated …

### PR DESCRIPTION
**Pull request by EricTran5791**
…the .gitignore

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "conartist",
+  "name": "conartist-root",
```
Comment (OinkIguana) [inline package.json]: why is this renamed?

Comment (EricTran5791) [reply package.json]: You can't have a child package with the same name. For example, your web-elm folder package name is also conartist. You'll get an npm dependency error.
`ERR! Refusing to install conartist as a dependency of itself`

Comment (OinkIguana) [reply package.json]: wat since when
